### DISCUSSION TITLE
Receiving "Error: no such module: fts5" on web

### DISCUSSION THREAD

**Discussion by jmolins** [simolus3/drift#2271] [Q&A]
Hi,

I'm using the fts5 module. It works properly on Android and iOS. But I receive the error "Error: no such module: fts5" on the web.

The web app is working OK without this module using the same configuration as in the "flutter_web_worker_example".

FTS5 is working OK as well on Android by adding the "sqlite3_flutter_libs" package. But I see that this package is not supported on web.

So my question is ¿Is FTS5 supported on the web build?

Maybe I'm missing some type of configuration or addition.

Thanks a lot

Jose

**Comment by simolus3** [comment]
I assume you're using the version based on SQL.js? If so, you will need to compile a version of that library that has FTS5 enabled, similar to how it was described in  [this issue](https://github.com/sql-js/sql.js/issues/392#issuecomment-633743691).

**Comment by jmolins** [comment]
Hi, Simon.
Your answer put me in the right way. I have changed to use the new drift/wasm.dart backend and it is working right now.
It has been easier than recompiling the sql.js
Thanks a lot
Jose
